### PR TITLE
Add Alchemy Devnet RPC Endpoint

### PR DESCRIPTION
**Pull request by avasisht23**
Hey there! Ajay from the Alchemy team. Given our new Solana support, I added the latest Alchemy devnet public RPC Url for Solana. Pls let me know if there are any questions, appreciate what you guys are doing at SOL Playground :) I'm definitely using your product for our tutorials!

### CODE REVIEW

**Review by acheroncrypto — CHANGES_REQUESTED** [review]
It would be better if these entries lived under the existing devnet endpoints, it will group that way in the UI too.

**Comment by acheroncrypto** [comment]
Thank you and great to hear! Will merge once you make the changes.

> I'm definitely using your product for our tutorials!
Playground will have tutorial section soon, let me know if you are interested!

**Review by acheroncrypto — APPROVED** [review]

**Comment by avasisht23** [comment]
Hi! Wondering if the beta.solpg.io website was updated after this merge? Noticing Alchemy is not an option in this dropdown yet.

image

**Pull request merged**
